src/serializers/html.js

```diff
@@ -108,10 +109,21 @@ class Html {
         return memo
       }
 
-      const block = {
-        kind: 'block',
-        type: this.defaultBlockType,
-        nodes: [node]
+      let block
+      if (this.defaultBlockType.constructor.name === 'String') {
```

Comment:
Can you change this to use `typeof === 'string'` instead of `.constructor.name === 'String'`? Thanks